packages/types/src/lib/types.ts

```diff
@@ -123,6 +123,7 @@ export type LITChain<T> = {
 
 export type LIT_NETWORKS_KEYS =
   | 'cayenne'
+  | 'datil-dev'
```

Comment:
💖  :D See above comments where we duplicated this union in the contracts sdk -- we should use this instead :)